airflow/www_rbac/views.py

```diff
@@ -224,27 +237,34 @@ def dag_stats(self, session=None):
             session.query(ds.dag_id, ds.state, ds.count)
         )
 
+        filter_dag_ids = appbuilder.sm.get_accessible_dag_ids()
```

Comment:
in general it would be better to filter on the database side, with something like:
```
if filter_dag_ids:
    qry = qry.filter(ds.in_(filter_dag_ids))
```

Reply:
thanks